src/components/ui/DatePicker/DatePicker.svelte

```diff
@@ -343,15 +344,13 @@
   }
 
   .date-picker-actions .action .action-icon {
-    display: inline-block;
+    display: flex;
     margin-right: 7px;
     text-align: right;
     width: 16px;
   }
 
   .date-picker-actions .action .action-label {
     display: inline-block;
-    font-weight: 400;
```

Comment:
Is the weight okay to remove? The design has a 500 weight (not sure why I set it to 400 in the first place 🤦🏻)

Reply:
Removed it since I added the `st-typography-body` class which has font weight of 400, was trying to avoid making any design changes here. But could add this back and bump to 500?

Reply:
ooh ok. sorry I didn't notice that. I'm good with this then 👍🏻